business_requirement/models/business.py

```diff
@@ -152,23 +147,26 @@ def _get_default_company(self):
     )
     portal_published = fields.Boolean("In Portal", default=False)
     user_id = fields.Many2one(
-        "res.users",
+        comodel_name="res.users",
         string="Owner",
         default=lambda self: self.env.user,
         required=True,
-        track_visibility="always",
+        tracking=True,
```

Comment:
If this is migrated from 13.0 this should be fixed on that version too. Because it is probably having no effect

Reply:
Thanks for creating the PR for v13, when it is merged I will add it here, reducing the difference of changes.